tests/PriorityMapTest.php

```diff
@@ -144,4 +163,38 @@ public function testSet_ObjectKey()
         $list->rewind();
         $this->assertEquals(41, $list->current());
     }
+
+    public function testSetDescOrder()
+    {
+        $key = new \stdClass();
+
+        $list = new PriorityMap();
+        $list->setDescOrder();
+
+        $list->set($key, 42, 1);
+        $list->set($key, 41, 0);
+        $list->set($key, 43, 2);
+
+        $this->assertSame(43, $list->current());
+    }
+
+    public function testToArray()
+    {
+        $key1 = 'k1';
+        $key2 = 'k2';
+        $key3 = 'k3';
+
+        $list = new PriorityMap();
+
+        $list->set($key1, 42, 1);
+        $list->set($key2, 41, 0);
+        $list->set($key3, 43, 2);
+
+        $result = $list->toArray();
+
+        $this->assertSame(array(
```

Comment:
https://www.php-fig.org/psr/psr-2/

Argument lists MAY be split across multiple lines, where each subsequent line
is indented once. When doing so, the first item in the list MUST be on the
next line, and there MUST be only one argument per line.

Reply:
run `composer check-style` to check

Reply:
I have run the ```check-style``` and ```fix-style``` before pushing the commits.